services/shhext/service_test.go

```diff
@@ -231,6 +231,51 @@ func (s *ShhExtSuite) TestRequestMessages() {
 	s.Contains(api.service.tracker.cache, common.BytesToHash(hash))
 }
 
+func (s *ShhExtSuite) TestDebugPostSync() {
+	mock := newHandlerMock(1)
+	s.services[0].tracker.handler = mock
+	s.nodes[0].Server().AddPeer(s.nodes[1].Server().Self())
+	symID, err := s.whisper[0].GenerateSymKey()
+	s.NoError(err)
+	client, err := s.nodes[0].Attach()
+	s.NoError(err)
+	var hash common.Hash
+
+	var testCases = []struct {
```

Comment:
It would be cool to test other cases:
- `errEnvelopeExpired`,
- context cancel/timeout using `client.CallContext`.

Reply:
I've been trying to cover `errEnvelopeExpired` but wasn't really able to do it. Context is covered now.

Reply:
We can add an issue for that. What about a situation when there are no Whisper peers? Shouldn't the envelope expire in such a case (plus setting TTL to 1 second)?

Reply:
awesome @adambabik! `ttl=1 + no whisper peers` fired that error, all lines covered now :)